cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -267,20 +341,40 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         className: 'edit-settings-timed-examination',
         events : {
             'change #id_not_timed': 'notTimedExam',
-            'change #id_timed_exam': 'showTimeLimit',
-            'change #id_practice_exam': 'showTimeLimit',
-            'change #id_proctored_exam': 'showTimeLimit',
+            'change #id_timed_exam': 'timedExam',
```

Comment:
Nit: I find these new names confusing as they don't imply an action. Can you add ``show`` back to the front of them, or if they aren't showing then use something like ``setTimedExam``.